microscope/devices.py

```diff
@@ -253,7 +253,10 @@ def initialize(self):
 
     def shutdown(self):
         """Shutdown the device for a prolonged period of inactivity."""
-        self.disable()
+        try:
+            self.disable()
+        except Exception as e:
+            _logger.debug("Exeption in disable() during shutdown: %s" % e)
```

Comment:
We are effectively ignoring an error during disable. I guess this should at least be of warning level and not debug?

Reply:
We're ignoring and error during disable because there's nothing else we can do - we're shutting it down. Warning is probably the more appropriate level.

Reply:
Fixed - log level changed to warn in 78fe9c8b9eb8d50bbd0fd0d2060cdd63417774ed.